addons/account/controllers/onboarding.py

```diff
@@ -0,0 +1,25 @@
+from odoo import http
+from odoo.http import request
+
+
+class Controller(http.Controller):
+
+    @http.route('/account/onboarding_invoice', auth='user')
+    def onboarding_invoice(self):
+        """ Returns the `banner` for the invoice onboarding panel.
+            It can be empty if the user has closed it or if he doesn't have
+            the permission to see it. """
+
+        if not request.env.user._is_admin() or request.env.user.company_id.onboarding_account_invoice_closed:
+            return ""
+
+        return request.render('account.onboarding_invoice_panel', {
+            'theme_color': 'blue',
+            'folded': request.env.user.company_id.onboarding_account_invoice_folded,
+            'close_method': 'close_onboarding_account_invoice',
```

Comment:
static values

Reply:
I don't understand this comment: the values are static in this case but will be different (and also static) for another onboarding view.